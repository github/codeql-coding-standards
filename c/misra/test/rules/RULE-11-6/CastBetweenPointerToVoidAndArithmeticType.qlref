rules/RULE-11-6/CastBetweenPointerToVoidAndArithmeticType.ql