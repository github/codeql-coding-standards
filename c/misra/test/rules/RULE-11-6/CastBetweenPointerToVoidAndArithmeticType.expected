| test.c:3:14:3:23 | (void *)... | Cast performed between a pointer to void type and an arithmetic type. |
| test.c:5:8:5:17 | (void *)... | Cast performed between a pointer to void type and an arithmetic type. |
| test.c:7:8:7:14 | (int)... | Cast performed between a pointer to void type and an arithmetic type. |
